composer/datasets/webdataset.py

```diff
@@ -31,7 +31,13 @@ def require_webdataset():
 
 
 def create_webdataset_meta(split_dir: str, n_samples: int, n_shards: int) -> None:
-    """Write a WebDataset meta file."""
+    """Write a WebDataset meta file.
```

Comment:
which ones of the below are public functions vs private? If Private, should lead with underscore.